pyiceberg/table/__init__.py

```diff
@@ -132,7 +132,8 @@
 )
 from pyiceberg.utils.concurrent import ExecutorFactory
 from pyiceberg.utils.config import Config
-from pyiceberg.utils.deprecated import deprecated, deprecation_message
+from pyiceberg.utils.deprecated import deprecated
+from pyiceberg.utils.deprecated import deprecation_message as deprecation_message
```

Comment:
```suggestion
from pyiceberg.utils.deprecated import deprecated, deprecation_message
```

Reply:
Unfortunately I think this is what the linter is suggesting 🙂 I'll revert these changes when I'm back at the desk

Reply:
Ah, I wasn't aware of that, sorry! 👍